src/slider/definitions.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
@imranhsayed Good thought - but lets remove this - it will make things more scalable, especially since we are getting stuff from JSON

Reply:
@junaidbhura When we remove this, we will have to use data type as `any`. Is that ok?

Reply:
@imranhsayed Lets do `{ [key: string]: any; }` where needed?